sqlmesh/core/dialect.py

```diff
@@ -267,8 +267,11 @@ def _parse_props(self: Parser) -> t.Optional[exp.Expression]:
         )
     else:
         value = self._parse_bracket(self._parse_field(any_token=True))
-
-    return self.expression(exp.Property, this=key.name.lower(), value=value)
+    name = key.name.lower()
+    if name == "path" and value:
+        # Make sure if we get a windows path that it is converted to posix
+        value = exp.Literal.string(value.this.replace("\\", "/"))
```

Comment:
pathlib.path(...).as_posix() or something rather than regex

Reply:
It doesn't work like that. 
```
pathlib.Path("..\..\test.py").as_posix()
> '..\\..\test.py'
```

That approach only works if you are on Windows.